Spellcheck for User guide (#9389)
braze-inc/braze-docs: _docs/_user_guide/analytics/query_builder.md

## Data and results

> Results, and exports of results, are tables that can contain up to 1,000 rows. For reports that require larger amounts of data, use another tool such as [Currents]({{site.baseurl}}/user_guide/data/braze_currents/) or Braze [export APIs]({{site.baseurl}}/api/endpoints/export).

Reviewer: ```suggestion
Results, and exports of results, are tables that can contain up to 1,000 rows. For reports that require larger amounts of data, you can use tools such as [Currents]({{site.baseurl}}/user_guide/data/braze_currents/) or the [export API endpoint]({{site.baseurl}}/api/endpoints/export).
```
